### DISCUSSION TITLE
Github sponsor link ?

### DISCUSSION THREAD

**Discussion by simonmichael** [vslinko/obsidian-zoom#91] [General]
I fricking love this plugin and use it constantly. Thanks for making it ! :)

I wanted to make a small donation and didn't see how. Github sponsors is usually the lowest friction and lowest fee for me, consider enabling that ?

**Comment by vslinko** [comment]
Hi @simonmichael 

Really thank you for the feedback!

I had donations, but now I can't accept them since I'm in Russia.
